app/src/androidTest/java/com/github/multimatum_team/multimatum/util/MockGroupRepository.kt

```diff
@@ -56,6 +56,14 @@ class MockGroupRepository(initialContents: List<UserGroup>) : GroupRepository()
         groups.remove(id)
     }
 
+    override suspend fun rename(id: GroupID, newName: String) {
+        groups[id] = groups[id]!!.copy(name = newName)
+    }
+
+    override suspend fun invite(id: GroupID, email: String) {
+        throw NotImplementedError("MockGroupViewModel.invite is not implemented")
```

Comment:
Sorry to insist on exceptions, but `NotImplementedError` suggests that the method is not *yet* implemented. Here `UnsupportedOperationException` would be clearer in my opinion.

Reply:
Yeah that makes sense, I don't think any tests will be required on the invite feature.